src/cephadm/cephadm.py

```diff
@@ -56,6 +56,7 @@
     LOG_DIR_MODE,
     SYSCTL_DIR,
     UNIT_DIR,
+    DAEMON_START_EXCEPTION_RETURN_CODE,
```

Comment:
not the most pithy var name ever, but ok

Reply:
It's not great. Do you have any good ideas for the name?

Reply:
not significantly better but `DAEMON_FAILED_ERROR`?  dunno. I'm fine if you don't want to change it.
Perhaps if you *do* move it to a common place eventually, we should make it an IntEnum so that we could namespace it and that enum's name would be helpful? Something like `CephadmErrorCodes.DAEMON_FAILED`?

Reply:
Makes sense. Given I plan to look into moving some of this stuff to a common location in the next week or so I think I'll just leave it for now and then just have it be changed when we swap to an enum.

Reply:
changed the name to DAEMON_FAILED_ERROR for now. Will revisit once the PR to put the default image names in a common location is merged.